Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidgetUtils.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/IO/Path/Path.h>
+#include <QWidget>
+#include <optional>
```

Comment:
The include that should be used here is <AzCore/std/optional.h>` since AZStd::optional is being used.
```suggestion
#include <AzCore/std/optional.h>
```